arxlive_spotlight_annotation/src/node_modules/es/bulk.mjs

```diff
@@ -0,0 +1,38 @@
+import * as _ from 'lamb'
+
+const generateBulkPayload = (operation, index) => _.pipe([
```

Comment:
for consistency I'd use `(method, index)` or conversely line 26 `const generate = generateBulkPayload(method, index)` could be `const generate = generateBulkPayload(operation, index)`